luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/SingleDataSourcePhysicalTableFactory.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.config.luthier.factories;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.yahoo.bard.webservice.config.luthier.Factory;
+import com.yahoo.bard.webservice.config.luthier.LuthierIndustrialPark;
+import com.yahoo.bard.webservice.config.luthier.LuthierValidationUtils;
+import com.yahoo.bard.webservice.data.config.LuthierPhysicalTableParams;
+import com.yahoo.bard.webservice.data.config.LuthierTableName;
+import com.yahoo.bard.webservice.data.dimension.DimensionColumn;
+import com.yahoo.bard.webservice.data.metric.MetricColumn;
+import com.yahoo.bard.webservice.data.time.DefaultTimeGrain;
+import com.yahoo.bard.webservice.table.ConfigPhysicalTable;
+
+import org.joda.time.DateTimeZone;
+
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.Locale;
+
+/**
+ * A factory that is used by default to support Simple (non-Composite) Physical Table.
+ * Classes that extend this abstract class:
+ *
+ * PermissivePhysicalTableFactory
+ * StrictPhysicalTableFactory
+ */
+abstract class SingleDataSourcePhysicalTableFactory implements Factory<ConfigPhysicalTable> {
+    private static final String ENTITY_TYPE = "single data source physical table";
+
+    /**
+     * Build the parameter for the subclass of SingleDataSourceParams to use.
+     *
+     * @param name  name of the LuthierTable as a String
+     * @param configTable  ObjectNode that points to the value of corresponding table entry in config file
+     * @param resourceFactories  should be the industrial park that needs to be paseed along to the builders
```

Comment:
Typo. `paseed` should be `passed`. Also, I would rephrase this as something like `@param resourceFactories  Source of the this table's dependencies (metrics, dimensions, etc)`

Reply:
Agreed.